crates/db_views/site/src/impls.rs

```diff
@@ -43,3 +45,26 @@ impl SiteView {
     Person::read(pool, site_view.local_site.multi_comm_follower).await
   }
 }
+
+pub fn user_backup_list_to_user_settings_backup(
+  local_user_view: LocalUserView,
+  lists: UserBackupLists,
+) -> UserSettingsBackup {
+  let vec_into = |vec: Vec<_>| vec.into_iter().map(Into::into).collect();
+
+  UserSettingsBackup {
+    display_name: local_user_view.person.display_name,
+    bio: local_user_view.person.bio,
+    avatar: local_user_view.person.avatar.map(Into::into),
+    banner: local_user_view.person.banner.map(Into::into),
+    matrix_id: local_user_view.person.matrix_user_id,
+    bot_account: local_user_view.person.bot_account.into(),
+    settings: Some(local_user_view.local_user),
+    followed_communities: vec_into(lists.followed_communities),
+    blocked_communities: vec_into(lists.blocked_communities),
+    blocked_instances: lists.blocked_instances,
+    blocked_users: lists.blocked_users.into_iter().map(Into::into).collect(),
+    saved_posts: lists.saved_posts.into_iter().map(Into::into).collect(),
+    saved_comments: lists.saved_comments.into_iter().map(Into::into).collect(),
```

Comment:
You can also use `vec_into` for these last ones.

Reply:
Whoops, merged before. I'll fix those now.